rpm/centos-7/docker-ce.spec

```diff
@@ -1,217 +0,0 @@
-Name: docker-ce
-Version: %{_version}
-Release: %{_release}%{?dist}
-Epoch: 3
-Summary: The open-source application container engine
-Group: Tools/Docker
-License: ASL 2.0
-Source0: engine.tgz
-Source1: cli.tgz
-URL: https://www.docker.com
-Vendor: Docker
-Packager: Docker <support@docker.com>
-
-# DWZ problem with multiple golang binary, see bug
-# https://bugzilla.redhat.com/show_bug.cgi?id=995136#c12
-%global _dwz_low_mem_die_limit 0
-%global is_systemd 1
-%global with_selinux 1
-
-BuildRequires: make
-BuildRequires: cmake
-BuildRequires: gcc
-BuildRequires: git
-BuildRequires: glibc-static
-BuildRequires: libtool-ltdl-devel
-BuildRequires: libseccomp-devel
-BuildRequires: device-mapper-devel
-BuildRequires: btrfs-progs-devel
-BuildRequires: pkgconfig(systemd)
-BuildRequires: pkgconfig(libsystemd-journal)
```

Comment:
This `BuildRequires` is only in the centos spec, but not in the Fedora spec (I seem to recall this package is no longer available in newer distros); something we'll have to take into account once we're splitting out Fedora/CentOS specs

Reply:
Yeah I also don't believe we actually end up using these anymore either since we don't build the daemon as part of the package